src/main/java/teammates/ui/pagedata/InstructorFeedbackResultsPageData.java

```diff
@@ -1781,4 +1785,28 @@ private int generateIndexForUser(String name, int index, Map<String, Integer> us
         return index + 1;
     }
 
+    /**
+     * Checks if we should show No Specific Section panel.
```

Comment:
Checks if any entity (student/instructor) belongs to no specific setion.

A side note: The method doesn't touch anything about panel and therefore the javadoc summary should not talk about panel also.